### ISSUE TITLE
Add support for gateways

### ISSUE THREAD

**Issue by kschiffer** [TheThingsNetwork/lorawan-devices#124]
#### Summary
The device repository should also allow sourcing gateway information.

#### Why do we need this?
To expand use cases of the DR to gateways. E.g. simplifying the gateway onboarding in the TTS Console: https://github.com/TheThingsNetwork/lorawan-stack/issues/3947

#### What is already there? What do you see now?
Support for end devices and a lot of already sourced data.

#### What is missing? What do you want to see?
Support for gateways to allow sourcing such data as well.

#### How do you propose to implement this?
Complementary to how end devices are implemented.

#### Can you do this yourself and submit a Pull Request?
This is one for @johanstokking

**Comment by johanstokking** [comment]
What do we want to put in the schema?

Suggesting:

- Band (to filter frequency plans)
- Radio settings for one or more SX1301s
- Supported protocols; UDP, Basic Station

**Comment by KrishnaIyer** [comment]
* Concentrator type (SX1301/SX1302)
* `clksrc`

**Comment by johanstokking** [comment]
Yes, we have quite some things to do in our other components to consume that, but this will be done.

**Comment by wireless-road** [comment]
Hello. Any updates on this? 

P.S. Do you have idea how following device might be classified:
https://thumb.cloud.mail.ru/weblink/thumb/xw1/Gcra/x1sJDaxsJ
There is mini PCI express form factor card that implements functionality a fully functional 8 channel LoRaWAN gateway's LoRa physics. Designed to integrate LoRaWAN gateway functionality to existing devices/solutions. 
Does it possible to place such device at your website?

**Comment by johanstokking** [comment]
No updates yet; we start with the new end device onboarding flow in The Things Stack and after that we'll work on the gateway onboarding flow which may come with gateway definitions in here.

As for your product; where would it be listed?

If you have documentation available to configure the gateway for The Things Stack, you can contribute to the documentation that is shown here: https://www.thethingsindustries.com/docs/gateways/. See the Edit on GitHub link in the footer to start contributing.
